### ISSUE TITLE
new feature: thoughts on honeypot & darknet sensor deployments

### ISSUE THREAD

**Issue by clayball** [madirish/hector#5]
I'm thinking of creating the honeypot(s) and darknet sensor(s) as small virtual machines that can be deployed via vagrant. Perhaps this can be git controlled so any updates to the honeypots repo can be pulled if/when desired. Thoughts?

**Comment by clayball** [comment]
I'm still on the fence about this. I do like the idea of things being modular and optional.

**Comment by madirish** [comment]
Yeah, the honeypots and darknet sensors really integrate via OSSEC so they can be deployed however you want - but I think the idea of pre-packaging them is great!  The only trick would be getting OSSEC to work properly (communicating back to the HECTOR host).
